Gemfile

```diff
@@ -30,7 +30,7 @@ gem 'tweetstream', '~> 2.6.1' # no longer maintained?
 gem 'twitter', '~> 5.5'
 gem 'flickraw', '~> 0.9.9'
 gem 'active_scaffold', '~> 3.5.0'
-gem 'active_scaffold_export', github: 'naaano/active_scaffold_export'
+gem 'active_scaffold_export', git: 'https://github.com/naaano/active_scaffold_export'
```

Comment:
@peggles2 , I updated this per the error message generated during `bundle install`:

The git source `git://github.com/naaano/active_scaffold_export.git` uses the `git` protocol, which transmits data without encryption. Disable this warning with `bundle config git.allow_insecure true`, or switch to the `https` protocol to keep your data secure.

Reply:
would it be possible to use https protocol?

Reply:
My change does use https:
`gem 'active_scaffold_export', git: 'https://github.com/...`